### PR TITLE
Add new naas group with kubeconfig subcommand

### PR DESCRIPTION
**Pull request by thokra-nav**
This command will use Google APIs to fetch cluster credentials and create or update a kubeconfig file.

It fetches GCP projects by labels defined by NAAS and iterates on all clusters in those projects.

Authentication is setup using the new third party authentication flow required by newer kubectls. It also configures it with an environment value to force authentication using the email that created the kubeconfig.

### CODE REVIEW

File: cmd/root/naas/config.go [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+import (
+	"fmt"
+
+	"github.com/nais/cli/cmd"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+var postgresCommand = &cobra.Command{
```
Comment (mortenlj) [inline cmd/root/naas/config.go:11]: Trenger et bedre navn på denne variabelen :slightly_smiling_face:

**Review by mortenlj — APPROVED** [review]
Lite å klage på her, bare en liten ting :+1:

**Comment by thokra-nav** [comment]
Mangler `management` cluster for oss, bør legge inn flagg for det også (eller defaulte hvis man har `--tenant *`)

**Comment by jhrv** [comment]
heier på denne

**Pull request merged**